python/doc/examples/data_analysis/smoothing_mixture.ipynb

```diff
@@ -0,0 +1,524 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Sensitivity of bandwidth in kernel smoothing\n",
+    "\n",
+    "## Introduction\n",
+    "\n",
+    "We consider the distribution\n",
+    "\n",
+    "$$\n",
+    "f_1(x) = w_1 f_A(x) + w_2 f_B(x)\n",
+    "$$\n",
+    "\n",
+    "for any $x\\in\\mathbb{R}$ where $f_A$ is the density of the gaussian distribution \n",
+    "$\\mathcal{N}(0,1)$, $f_B$ is the density of the gaussian distribution \n",
+    "$\\mathcal{N}(3/2,(1/3)^2)$ and the weights are $w_1 = \\frac{3}{4}$ and $w_2 = \\frac{1}{4}$.\n",
+    "\n",
+    "This is a mixture of two gaussians: 1/4th of the observations have the $\\mathcal{N}(0,1)$ \n",
```

Comment:
```suggestion
    "for any $x\\in\\mathbb{R}$ where $f_A$ is the density of the Normal distribution \n",
    "$\\mathcal{N}(0,1)$, $f_B$ is the density of the Normal distribution \n",
    "$\\mathcal{N}(3/2,(1/3)^2)$ and the weights are $w_1 = \\frac{3}{4}$ and $w_2 = \\frac{1}{4}$.\n",
    "\n",
    "This is a mixture of two Normal distributions: 1/4th of the observations have the $\\mathcal{N}(0,1)$ \n",
```
@adutfoy advocated for more consistency in our vocabulary: _Normal_ distributions and _Gaussian_ processes